env_support/cmake/custom.cmake

```diff
@@ -47,6 +48,14 @@ if(NOT LV_CONF_BUILD_DISABLE_THORVG_INTERNAL)
     endif()
 endif()
 
+if(NOT LV_CONF_BUILD_DISABLE_LZ4_INTERNAL)
```

Comment:
This block is about when INTERNAL is not disabled, so internal is enabled. Why do we need any kind of CMake update in this case? Those 2 lz4 files will be `glob`ed  with all the other LVGL files.

Reply:
draw/image_format_lz4_compressed_rotate.png